lilypad/server/models/generations.py

```diff
@@ -0,0 +1,62 @@
+"""Generations table and models."""
+
+from typing import TYPE_CHECKING, Optional
+from uuid import UUID
+
+from sqlalchemy import JSON, Column
+from sqlmodel import Field, Relationship, SQLModel
+
+from ..._utils import Closure, DependencyInfo
+from .base_organization_sql_model import BaseOrganizationSQLModel
+from .table_names import (
+    GENERATION_TABLE_NAME,
+    PROJECT_TABLE_NAME,
+    PROMPT_TABLE_NAME,
+)
+
+if TYPE_CHECKING:
+    from .projects import ProjectTable
+    from .prompts import PromptTable
+    from .spans import SpanTable
+
+
+class _GenerationBase(SQLModel):
+    """Base Generation Model."""
+
+    project_uuid: UUID | None = Field(
+        default=None, foreign_key=f"{PROJECT_TABLE_NAME}.uuid"
+    )
+    prompt_uuid: UUID | None = Field(
+        default=None, foreign_key=f"{PROMPT_TABLE_NAME}.uuid"
+    )
+    name: str = Field(nullable=False, index=True, min_length=1)
+    signature: str = Field(nullable=False)
+    code: str = Field(nullable=False)
+    hash: str = Field(nullable=False, index=True, unique=True)
+    dependencies: dict[str, DependencyInfo] = Field(
+        sa_column=Column(JSON), default_factory=dict
+    )
+
+
+class GenerationCreate(Closure):
+    """Generation create model."""
+
+    arg_types: dict[str, str]
```

Comment:
`arg_types` is only on create, but not on the GenerationTable so this column will never be created. is this intended?

Reply:
done